### ISSUE TITLE
Piano mode

### ISSUE THREAD

**Issue by Andreya-Autumn** [surge-synthesizer/conduit#54]
Aka "reuse active" mode. Seems like the nicer behavior here where we only get one. Currently it is stacking.

**Comment by baconpaul** [comment]
Yeah this will come with the voice manager work. Right now we are multi omni poly only.

**Comment by elanhickler** [comment]
Is the desired behavior, that when you play a C3 for example, then later play C3 again, that the C3 voice is reset and/or retriggered? I would implement a retrigger or a reset as user options for that.

**Comment by baconpaul** [comment]
Yeah piano mode is a sue name for exact that do two key presses with log release make one voice (piano mode) or 2 (polysynth ode)
